### DISCUSSION TITLE
Share links with Obtainium

### DISCUSSION THREAD

**Discussion by Cristofer07** [ImranR98/Obtainium#1610] [Ideas]
How about adding that links can be shared with this app? to speed up adding apps a little, instead of copying and pasting, just share and add

**Comment by ImranR98** [comment]
This is already a thing. You can share links by long pressing on the app in the apps list, tapping the "more" icon, and picking "Share app configuration as HTML link".

There are also a lot of crowdsourced configs (with direct links) at https://apps.obtainium.imranr.dev/

**Reply by Cristofer07** [reply]
Mmmmm I think I'm not explaining myself well, I'm talking about being able to go from the browser to a page like fdroid or apkpure and from the browser menu being able to share the page with obtainium, and that by sharing it with obtainium, the application is added to your apps, more than anything I say to avoid copying and pasting every application you want to add, because if you want to add many applications this becomes somewhat annoying

**Comment by DarkCrypt** [comment]
Issue #109.
